src/process.h

```diff
@@ -53,6 +53,8 @@ namespace proc {
     std::string id;
     bool elevated;
     bool auto_detach;
+    bool wait_all;
+    std::chrono::seconds exit_timeout;
```

Comment:
Should the comment above this be updated?

`That is why some applications need be run and forgotten about`

Reply:
Yep, updated it.